docs/source/backend.rst

```diff
@@ -78,19 +122,19 @@ Sox IO Backend
 
 ``sox_io`` backend is available on ``torchaudio`` installation with C++ extension. It is currently not available on Windows system.
 
-This new backend is recommended over ``sox`` backend. You can switch from another backend to ``sox_io`` backend with the following;
+This new backend is recommended over ``sox`` backend, and will become the default backend in ``0.8.0`` release when C++ extension is present.
+
+You can switch from another backend to ``sox_io`` backend with the following;
 
 .. code::
 
    torchaudio.set_audio_backend("sox_io")
 
-The function call to this backend can be Torchsript-able. You can apply :func:`torch.jit.script` and dump the object to file, then call it from C++ application.
+The function calls to this backend are TorchSript-able. You can apply :func:`torch.jit.script` and dump the object toa file, then call it from C++ application.
```

Comment:
Probably not for this PR, but this might be a good opportunity to elaborate with an example, or point the reader to another reference.